src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -20,20 +20,22 @@ export interface PiletInjectorConfig extends KrasInjectorConfig {
 export default class PiletInjector implements KrasInjector {
   public config: PiletInjectorConfig;
   private port: number;
+  private https: boolean;
```

Comment:
Alternatively, call this `protocol` and have it set to either `http` or `https` -> then we can spare the conditional below and evaluate it only once.

Reply:
ok